OTAcceleratorCore/OTMultiPartyCommunicator.m

```diff
@@ -242,29 +245,37 @@ - (NSError *)disconnect {
     }
 
     for (OTMultiPartyRemote *subscriberObject in self.subscribers) {
+        [loggingWrapper.logger logEventAction:KLogActionRemoveRemote
+                                                     variation:KLogVariationAttempt
+                                                    completion:nil];
         OTError *error = nil;
         OTSubscriber *subscriber = subscriberObject.subscriber;
         [subscriber.view removeFromSuperview];
         [self.session unsubscribe:subscriber error:&error];
         if (error) {
+            [loggingWrapper.logger logEventAction:KLogActionRemoveRemote
+                                        variation:KLogVariationFailure
+                                       completion:nil];
             NSLog(@"%s: %@", __PRETTY_FUNCTION__, error);
         }
         [subscriberObject.subscriberView removeFromSuperview];
         [subscriberObject.subscriberView clean];
         subscriberObject.subscriber = nil;
         subscriberObject.subscriberView = nil;
+        [loggingWrapper.logger logEventAction:KLogActionRemoveRemote
+                                    variation:KLogVariationSuccess
+                                   completion:nil];
```

Comment:
@Lucashuang0802 you are right, the error is not a return. I will add  the success case in the else.